index.bs

```diff
@@ -11126,13 +11126,18 @@ and is described in more detail in [[#interface-prototype-object]].
 Note: Since an [=interface object=] is a [=function object=]
 the <code>typeof</code> operator will return "function" when applied to an interface object.
 
+An interface may have <dfn export>overridden constructor steps</dfn>, which can
+change the behavior of the [=interface object=] when called or constructed. By
+default interfaces do not have such steps.
+
```

Comment:
Maybe add a warning like we have a few already for legacy features, saying something like

> In general, constructors are described by defining a [=constructor operation=] and its behavior.
> The [=overridden constructor steps=] are used only for more complicated situations. 
> Editors who wish to use this feature are strongly advised to discuss this by filing an issue before proceeding.